articles/azure-monitor/app/tutorial-asp-net-core.md

```diff
@@ -0,0 +1,525 @@
+
+```json
+{
+  //... additional code removed for brevity
+  "Logging": {
+    "LogLevel": { // No provider, LogLevel applies to all the enabled providers.
+      "Default": "Warning"
+    },
+    "ApplicationInsights": { // Specific to the provider, LogLevel applies to the Application Insights provider.
+      "LogLevel": {
+        "ValuesController": "Error" //Log Level for the "ValuesController" category
+      }
+    }
+  }
+}
+```
+
+Deploying the sample application with the preceding code in *appsettings.json* will yield only the error trace being sent to Application Insights when interacting with the **ValuesController**. This is because the **LogLevel** for the **ValuesController** category is set to **Error**, therefore the **Warning** trace is suppressed.
+
+### Setting a default log level with code
+
+In **Program.cs**, locate the line of code: `var builder = WebApplication.CreateBuilder(args)` and add the following code to set the default logging level.
+
+```csharp
+var builder = WebApplication.CreateBuilder();
+builder.Logging.SetMinimumLevel(LogLevel.Warning);
+```
+
+### Create filter rules with code
+
+Filter rules may also be set in code. In **Program.cs**, locate the line of code: `var builder = WebApplication.CreateBuilder(args)` and add the following code to set the logging level for the **ValuesController** category specific to the **ApplicationInsightsLoggerProvider**.
+
+```csharp
+var builder = WebApplication.CreateBuilder(args);
+builder.Logging.AddFilter<Microsoft.Extensions.Logging.ApplicationInsights.ApplicationInsightsLoggerProvider>("ValuesController", LogLevel.Error);
+```
+
+## Turn off logging to Application Insights
+
+To disable logging using configuration, set all LogLevel values to "None".
+
+```json
+{
+  //... additional code removed for brevity
+  "Logging": {
+    "LogLevel": { // No provider, LogLevel applies to all the enabled providers.
+      "Default": "None"
+    },
+    "ApplicationInsights": { // Specific to the provider, LogLevel applies to the Application Insights provider.
+      "LogLevel": {
+        "ValuesController": "None" //Log Level for the "ValuesController" category
+      }
+    }
+  }
+}
+```
+
+Similarly, within code, set the default level for the `ApplicationInsightsLoggerProvider` and any subsequent log levels to **None**.
+
+```csharp
+var builder = WebApplication.CreateBuilder(args);
+builder.Logging.AddFilter<ApplicationInsightsLoggerProvider>("", LogLevel.None);
+builder.Logging.AddFilter<Microsoft.Extensions.Logging.ApplicationInsights.ApplicationInsightsLoggerProvider>("ValuesController", LogLevel.None);
+```
+
+## Open-source SDK
+
+* [Read and contribute to the code](https://github.com/microsoft/ApplicationInsights-dotnet).
+
+For the latest updates and bug fixes, see the [release notes](./release-notes.md).
+
+## Next steps
```

Comment:
need to mention the codeless approach as well. either here, or right at the beginning of the doc itself.

Reply:
Fixed.